build-release-tools/lib/PlatformClients.py

```diff
@@ -99,6 +99,8 @@ def release_box(self, atlas_version):
         resp = self.session.put(release_url)
         if resp.ok:
             print "Release version {0} successfully!".format(atlas_version)
+        elif resp.text.find("already") != -1 and resp.status_code == 422:
```

Comment:
I'm not sure if 422 is only for "already released" so add text-find double check.